provider/README.md

```diff
@@ -83,10 +83,10 @@ Data: `{ "trips": [] }`, an array of objects with the following structure
 
 | Field | Type    | Required/Optional | Comments |
 | ----- | -------- | ----------------- | ----- |
-| `provider_name` | String | Required | The public-facing name of the Provider |
 | `provider_id` | UUID | Required | A UUID for the Provider, unique within MDS |
+| `provider_name` | String | Required | The public-facing name of the Provider |
 | `device_id` | UUID | Required | A unique device ID in UUID format |
-| `vin` | String | Required | Vehicle Identification Number assigned by Provider |
+| `vehicle_id` | String | Required | Vehicle Identification Number assigned by Provider |
```

Comment:
for vehicle ID or VIN, we should probably add something like `must match the number/serial on physical device"

Reply:
Yes, I agree.   Go ahead and change.

Reply:
dbb2f4a